I Can't install library

I get the error when I install

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^16.8.0 || ^17" from @fluentui/react-component-event-listener@0.51.7
npm ERR!   node_modules/@fluentui/react-component-event-listener
npm ERR!     @fluentui/react-component-event-listener@"~0.51.6" from semantic-ui-react@2.0.3
npm ERR!     node_modules/semantic-ui-react
npm ERR!       semantic-ui-react@"^2.0.3" from the root project
npm ERR!       1 more (react-semantic-ui-datepickers)
npm ERR!   peer react@"^16.8.0 || ^17" from @fluentui/react-component-ref@0.51.7
npm ERR!   node_modules/@fluentui/react-component-ref
npm ERR!     @fluentui/react-component-ref@"~0.51.6" from semantic-ui-react@2.0.3
npm ERR!     node_modules/semantic-ui-react
npm ERR!       semantic-ui-react@"^2.0.3" from the root project
npm ERR!       1 more (react-semantic-ui-datepickers)
npm ERR!   16 more (@semantic-ui-react/event-stack, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8" from dayzed@3.2.1
npm ERR! node_modules/react-semantic-ui-datepickers/node_modules/dayzed
npm ERR!   dayzed@"3.2.1" from react-semantic-ui-datepickers@2.13.0
npm ERR!   node_modules/react-semantic-ui-datepickers
npm ERR!     react-semantic-ui-datepickers@"*" from the root project
```

--- a/package.json
+++ b/package.json
@@ -72,7 +72,7 @@
     "typescript": "4.2.4"
   },
   "peerDependencies": {
-    "react": ">=16.8.0",
+    "react": "^16.8 || ^17.0",
     "semantic-ui-react": ">=0.75.0"
   },
   "resolutions": {

--- a/stories/index.stories.tsx
+++ b/stories/index.stories.tsx
@@ -47,6 +47,7 @@ export const basicUsage = () => {
   const maxDate = new Date(date('Max date', new Date('2030-01-01')));
   const onlyWeekdays = boolean('Only weekdays (filterDate example)', false);
   const controlValue = boolean('Control value', false);
+  const error = boolean('Error state', false);
   const initialValue = controlValue
     ? type === 'basic'
       ? new Date(date('Initial value'))
@@ -64,6 +65,7 @@ export const basicUsage = () => {
         clearOnSameDateClick={clearOnSameDateClick}
         clearable={clearable}
         datePickerOnly={datePickerOnly}
+        error={error}
         filterDate={filterDate}
         firstDayOfWeek={firstDayOfWeek}
         format={format}

